keras_cv/models/backbones/mobilenet_v3/mobilenet_v3_backbone_presets.py

```diff
@@ -0,0 +1,153 @@
+# Copyright 2023 The KerasCV Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""MobileNetV3 model preset configurations."""
+
+from tensorflow.keras import layers
+
+import keras_cv.models.backbones.mobilenet_v3.mobilenet_v3_backbone as backbone
+
+
+def stack_fn_s(x, kernel, activation, se_ratio, alpha=1.0):
```

Comment:
Let's define these stack_fns in the backbone class itself, so that this presets file can hold just the preset info.

Reply:
These functions need more descriptive names

Reply:
Done